src/components/FilePreview.js

```diff
@@ -32,9 +32,18 @@ export default class FilePreview extends Component {
     }
     return (
       <div className="file-preview">
+        {
```

Comment:
I know I've put a few inline conditions in the front-end but it's better to pull them out of return statement.

```javascript
let theme_indicator;
let btn_delete;

if(file.from_theme) {
  theme_indicator = <div...
} else {
  btn_delete = <button...
}
```